File: ALGORITMOS-E-ESTRUTURAS-DE-DADOS-II/#Teorico/0 - Revisao/6 - Comandos de Repetição/repeat_comand.cs
using System;

class repeat_comand{
    public static void Main (string[] args){
        double base = 0, altura = 0, area = 0;

        for (int i = 0; base >= 0; i++){
            Console.WriteLine("Digite a base do {0}º retângulo: ", i);
            base = double.Parse(Console.ReadLine());

            if (base >= 0){
                Console.WriteLine("Digite a altura do " + i + "º retângulo: ");
                altura = double.Parse(Console.ReadLine());

                area = base * altura;

                Console.WriteLine("A área do {0}º retângulo = {1}. \n", i, area);
            }
        }
    }
}

File: AEDS II/codigos/0 - apresentacao-linguagens/repeat_comand.cpp
#include <iostream>

using namespace std;

int main(int argc, char const *argv[]){
    float base, altura, area;

    do{
        cout << "Digite a base do retângulo: ";
        cin >> base;

        if (base > 0){
            cout << "Digite a altura do retângulo: ";
            cin >> altura;

            area = base * altura;

            cout << "A área do retângulo = " << area << endl;
        }
        
    } while (base > 0);
    
    return 0;
}